packages/tooling/fast-tooling-react/src/form/custom-controls/css/color-picker/color-picker.ts

```diff
@@ -0,0 +1,376 @@
+    public readOnly: boolean;
+    private readOnlyChanged(): void {
+        if (this.proxy instanceof HTMLElement) {
+            this.proxy.readOnly = this.readOnly;
+        }
+    }
+
+    /**
+     * Indicates that this element should get focus after the page finishes loading. See {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#htmlattrdefautofocus | autofocus HTML attribute} for more information.
+     * @public
+     * @remarks
+     * HTML Attribute: autofocus
+     */
+    @attr({ mode: "boolean" })
+    public autofocus: boolean;
+    private autofocusChanged(): void {
+        if (this.proxy instanceof HTMLElement) {
+            this.proxy.autofocus = this.autofocus;
+        }
+    }
+
+    /**
+     * Sets the placeholder value of the element, generally used to provide a hint to the user.
+     * @public
+     * @remarks
+     * HTML Attribute: placeholder
+     * Using this attribute does is not a valid substitute for a labeling element.
+     */
+    @attr
+    public placeholder: string;
+    private placeholderChanged(): void {
+        if (this.proxy instanceof HTMLElement) {
+            this.proxy.placeholder = this.placeholder;
+        }
+    }
+
+    /**
+     * Flag indicating that the color UI is visible.
+     */
+    @observable
+    public open: boolean;
+
+    /**
+     * Flag indicating that the color UI is activily listening for mouse move and up events.
+     */
+    @observable
+    public mouseActive: boolean = false;
+
+    /**
+     * Object containing all of the properties displayed in the color picker UI.
+     */
+    @observable
+    public uiValues: ColorPickerUI = new ColorPickerUI(
+        new ColorRGBA64(1, 0, 0),
+        new ColorHSV(0, 1, 1)
+    );
+
+    /**
+     * A reference to the internal input element
+     * @internal
+     */
+    public control: HTMLInputElement;
+
+    /**
+     * A reference to the HTMLElement that is the current target of mouse move events.
+     */
+    private currentMouseTarget: HTMLElement = null;
+
+    /**
+     * A string indicating which of the three graphical elements is the current mouse target. ['sv','h','a']
+     */
+    private currentMouseParam: string;
+
+    /**
+     * The ColorRGBA64 representation of the current color value.
+     */
+    private currentRGBColor: ColorRGBA64;
+
+    /**
+     * The ColorHSV representation of the current color value.
+     */
+    private currentHSVColor: ColorHSV;
+
+    /**
+     * @internal
+     */
+    public connectedCallback(): void {
+        super.connectedCallback();
+        this.open = false;
+        if (!isNullOrWhiteSpace(this.value)) {
+            this.currentRGBColor = parseColor(this.value);
+        } else {
+            this.currentRGBColor = new ColorRGBA64(1, 0, 0, 1);
+        }
+        this.currentHSVColor = rgbToHSV(this.currentRGBColor);
+        this.updateUIValues(false);
+
+        this.proxy.setAttribute("type", "colorpicker");
```

Comment:
The type should be color - see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/color

```suggestion
        this.proxy.setAttribute("type", "color");
```

Reply:
Done.